src/main/java/com/ericsson/bss/cassandra/ecaudit/AuditEntryBuilderFactory.java

```diff
@@ -62,20 +73,49 @@
 
     public Builder createEntryBuilder(String operation, ClientState state)
     {
-        CQLStatement statement;
         try
         {
-            statement = QueryProcessor.getStatement(operation, state).statement;
+            CQLStatement statement = QueryProcessor.getStatement(operation, state).statement;
+            return createEntryBuilder(statement);
+        }
+        catch (KeyspaceNotDefinedException e)
```

Comment:
This seems to be specific to a keyspace, would we get into a similar scenario if a table does not exist?